modules/custom/reservation/reservation.routing.yml

```diff
@@ -25,3 +25,13 @@ reservation.MovieExporter:
     _title: 'Movie Exporter'
   requirements:
     _permission: 'administer site configuration'
+reservation.bookShowcase:
+  path: '/book-showcase'
+  defaults:
+    _controller: '\Drupal\reservation\Controller\ReservationController::bookContent'
```

Comment:
bookContent po meni ne doprinosi dovoljno informacija o tome sta funkcija radi, pa pogledaj kako to da preimenujes.